src/App/Main/Product/ProductListItem.js

```diff
@@ -1,181 +1,137 @@
-import React, { Component } from 'react'
-import PropTypes from 'prop-types'
-import './ProductListItem.css'
-import QuantityInput from '../../../Component/QuantityInput/QuantityInput'
-import { connect } from 'react-redux'
-import { Link } from 'react-router-dom'
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
 
-class ProductListItem extends Component {
-
-    static propTypes = {
-        name: PropTypes.string.isRequired,
-        description: PropTypes.string,
-        image: PropTypes.string,
-        type: PropTypes.string.isRequired,
-        screenSize: PropTypes.number,
-        capacity: PropTypes.number,
-        price: PropTypes.number.isRequired,
-    }
+import './ProductListItem.css';
 
-    static defaultProps = {
-        isLiked:false,
-    }
+import QuantityInput from '../../../Component/QuantityInput/QuantityInput';
 
-    state = {
-        productCount: 1
-    }
-
-    onDecrementClick = () => {
-        this.setState((prevState)=>({
-            productCount: prevState.productCount - 1
-        }))
-    }
-
-    onIncrementClick = () => {
-        this.setState((prevState)=>({
-            productCount: prevState.productCount + 1
-        }))
-    }
-
-    renderQuantity() {
-        return (
-            <QuantityInput 
-                quantity = {this.state.productCount}
-                onDecrementClick = {this.onDecrementClick}
-                onIncrementClick = {this.onIncrementClick}
-                minValue={1}
-                maxValue={9}
-            />
-        )
-    }
-
-    onLikeClick = () => {
-        if (this.props.isLiked) {
-            this.props.dispatchDislike(this.props.id)
-        } else {
-            this.props.dispatchLike(this.props.id)
-        }
+class ProductListItem extends Component {
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    description: PropTypes.string,
+    image: PropTypes.string,
+    type: PropTypes.string.isRequired,
```

Comment:
If prop is optional, add default value of prop in `defaultProps`. Add ESLint rule that will validate it